spec/controllers/collections_controller_spec.rb

```diff
@@ -21,26 +21,26 @@ def create_collection_call(route_id, collection_point_id, pocket_attributes)
 
       context 'when creating valid collections' do
         it 'does return success' do
-          create_collection_call(auth_route.id, collection_point.id, serial_numbers)
+          create_collection_call(auth_route.id, container.id, serial_numbers)
```

Comment:
Maybe you could avoid repeating the create_collection_call in this context, adding before(:each) { create_collection_call(auth_route.id, container.id, serial_numbers) }